README.md

```diff
@@ -126,14 +126,14 @@ If you use the Habitat platform in your research, please cite the [Habitat 1.0](
     python examples/example.py
```

Comment:
I must have forgotten to add my changes to the `README.md`. Nice catch.

You don't need to `cd` You should do just like in CI :
```
pip install -e habitat-lab
pip install -e habitat-baselines
```

If you really want to use `cd`, then it would look like

```
# pip install habitat-lab stable version
...
cd habitat-lab
pip install -e .
...

# Install habitat-baselines from source
...
cd habitat-baselines
pip install -e .
...
```

Reply:
Yeah, the `habitat-lab` in my snippets means repository root (after git clone).